frontend/src/pages/Main.tsx

```diff
@@ -43,7 +48,7 @@ const Main: FC = () => {
   function dateChange(date: any) {
     const YearMonth: YearMonth = {
       year: date.getFullYear(),
-      month: date.getMonth() + 1,
+      month: date.getMonth() + 1, // +1 why
```

Comment:
https://stackoverflow.com/questions/2552483/why-does-the-month-argument-range-from-0-to-11-in-javascripts-date-constructor, for the comment :))

Reply:
yeah i forgot to remove that while rebasing, i'll add a comment to explain it ty